scripts/tags/video.js

```diff
@@ -0,0 +1,140 @@
+(function() {
+  'use strict';
+  
+  /**
```

Comment:
we can remove this orphan function doc

Reply:
Unsure of what this is, sorry. image.js has this same type of function definition.